src/main/java/wromaciej/moviesrental/movies/model/Actor.java

```diff
@@ -1,38 +1,57 @@
 package wromaciej.moviesrental.movies.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
 
 @Entity
 @Table(name = "actor", schema = "public")
 public class Actor {
 
     @Id
     @Column(name = "actor_id")
-    private Integer actorId;
+    private Integer id;
 
     @Column(name = "first_name")
     private String firstName;
 
     @Column(name = "last_name")
     private String lastName;
 
-    public Actor(){}
+    @ManyToMany(cascade = {CascadeType.ALL})
+    @JoinTable(
+            name = "film_actor",
+            joinColumns = {@JoinColumn(name = "actor_id")},
+            inverseJoinColumns = {@JoinColumn(name = "film_id")}
+    )
+    private Collection<Movie> movies = new ArrayList<Movie>();
+
+    public Actor() {
+    }
 
     public Actor(Integer actorId, String firstName, String lastName) {
-        this.actorId = actorId;
+        this.id = actorId;
         this.firstName = firstName;
         this.lastName = lastName;
     }
 
+    @JsonIgnore
```

Comment:
getId and other actor getters are more important and should be before get/setMovies. Generally the best would be to have id getters/setters after constructors.

BTW. add to your bookmarks (toilet read :)) links about coding styles:
https://github.com/twitter/commons/blob/master/src/java/com/twitter/common/styleguide.md
https://google.github.io/styleguide/javaguide.html
http://geosoft.no/development/javastyle.html#Methods

there might be differences between them, but common things are for sure good practices ;)